app/controllers/idv/hybrid_mobile/socure/errors_controller.rb

```diff
@@ -10,23 +10,19 @@ class ErrorsController < ApplicationController
         include StepIndicatorConcern
         include SocureErrorsConcern
 
-        def show
-          error_code = error_code_for(handle_stored_result)
+        def show(error_code: nil)
```

Comment:
I didn't know you could do this! Since this is an action, should we check the params hash to get the error_code?
```suggestion
        def show
          error_code = params[:error_code]
```